.github/workflows/non-english-warning.yml

```diff
@@ -0,0 +1,45 @@
+name: Check for non-English updates outside of Crowdin
+
+on:
+  pull_request:
+    paths:
+      - "public/content/translations/**/*.md"
+      - "src/intl/**/*.json"
+      - "!src/intl/en/**
+
+jobs:
+  check_branch_name:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Exit early if branch name contains 'crowdin'
+        run: |
+          if [[ "${{ github.head_ref }}" == *crowdin* ]]; then
+            echo "Branch name contains 'crowdin', stopping workflow"
+            exit 1
+          fi
```

Comment:
Exits if `crowdin` is found in the branch name. The `utils.ts` update in this PR includes the addition of this string segment to the branch names generated in the Crowdin CI flow, so these will not be flagged.

Reply:
Exiting with `1` (a non-zero status) will be seen as a "failure" and cause the following job to not run.